sdk/openai/Azure.AI.OpenAI/README.md

```diff
@@ -694,6 +694,25 @@ Console.WriteLine($"Translation ({translation.Duration.Value.TotalSeconds}s):");
 Console.WriteLine(translation.Text);
 ```
 
+
```

Comment:
There is only one blank line b/w samples. Please remove the additional line.

Reply:
Resolve by https://github.com/ninghu/azure-sdk-for-net/pull/11/commits/de73afce474165f9b31c22fb85dee73ec798bb03